### DISCUSSION TITLE
Exporting into the future - Causing problems

### DISCUSSION THREAD

**Discussion by DrSpaldo** [blakeblackshear/frigate#11231] [General]
I've just tried to export and forgot to change the AM/PM for today. It has caused my exporting to hang (spinning wheel). There is nothing in the logs regarding exports or errors. 

I was wondering if there should be a check prior to allowing an export that the time entered is actually in the future? 

Also, secondary, is there a way to stop the current export? In particular if it is broken and not working, there does not appear to be a cancel option.

**Comment by NickM-27** [comment]
Future time is enforced in the new ui
